WooCommerce/Classes/ViewRelated/BottomSheet/ListSelector/BottomSheetListSelectorViewController.swift

```diff
@@ -20,7 +20,6 @@ UIViewController, UITableViewDataSource, UITableViewDelegate where Command.Model
         guard let tableView = tableView else {
             return .zero
         }
-        tableView.layoutIfNeeded()
```

Comment:
`contentSize` is called by the bottom sheet library to determine the size of the sheet before the sheet is displayed. Calling `layoutIfNeeded` in that case forces the table view to layout its subviews with inaccurate info about device size, causing issues with the long labels. Removing this fixes the issues for the labels, but the bottom sheet is presented with a smaller size initially (it can be dragged up to display the full size).

For further improvement, we can switch to using the native bottom sheet instead.

Reply:
Thanks for adding this explanation.